pkg/api/norman/customization/aks/listers.go

```diff
@@ -0,0 +1,202 @@
+package aks
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"regexp"
+	"sort"
+
+	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2020-09-01/containerservice"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-07-01/network"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/mcuadros/go-version"
+)
+
+type virtualNetworksResponseBody struct {
+	Name          string   `json:"name"`
+	ResourceGroup string   `json:"resourceGroup"`
+	Subnets       []subnet `json:"subnets"`
+}
+
+type subnet struct {
+	Name         string `json:"name"`
+	AddressRange string `json:"addressRange"`
+}
+
+var regex = regexp.MustCompile("/resourceGroups/(.+?)/")
+
+func NewClientAuthorizer(cap *Capabilities) (autorest.Authorizer, error) {
+	if cap.AuthBaseURL == "" {
```

Comment:
This and `BaseURL` below should already be set in `getCloudCredential` or `getCredentialsFromBody` so this shouldn't be needed

Reply:
I agree and removed